docs/reference/gnsf_dynamic_system_structure.tex

```diff
@@ -0,0 +1,62 @@
+\documentclass{report}
+
+\usepackage{amsmath}
+\usepackage{amssymb}
+\usepackage{color}
+
+\newcommand {\matr}[2]{\left[\begin{array}{#1}#2\end{array}\right]}
+
+\newcommand{\R}{{\mathbb{R}}}
+    
+\newcommand{\ind}[1]{{_{\text{#1}}}}
+\newcommand{\diag}[1]{\text{diag}\left( {#1} \right)}
+\newcommand{\upind}[1]{\ensuremath{{^{[\text{#1}]}}}}
+\newcommand{\uind}[1]{^{[#1]}}
+\newcommand{\defI}[1]{\textit{#1}}
+\newcommand{\nxone}{{n_{x_1}}}
+\newcommand{\nxtwo}{{n_{x_2}}}
+\newcommand{\nx}{\ensuremath{n_x}}
+%	\newcommand{\nxone}{{n_{x^{\text{[1]}}}}}
+%	\newcommand{\nxtwo}{{n_{x^{\text{[2]}}}}}
+\newcommand{\nz}{{n_{z}}}
+\newcommand{\nout}{n_{\text{out}}}
+\newcommand{\nin}{n_{\text{in}}}
+\newcommand{\ninx}{n_{y}}
+\newcommand{\ninu}{n_{\hat{u}}}
+%\newcommand{\dtAnf}[1]{\glqq#1\grqq}
+\newcommand{\total}[0]{\text{d}}
+\newcommand{\DD}[0]{\text{D}}
+\newcommand{\dtotal}[2]{\frac{\text{d} {#1}}{\text{d} {#2} }}
+\newcommand{\dpartial}[2]{\frac{\partial {#1}}{\partial {#2} }}
+
+\begin{document}
+
+\chapter*{Generalized Nonlinear Static Feedback Structure}
+\newcommand{\axdot}{E_{11}}
+\newcommand{\ax}{A_{1}}
+\newcommand{\au}{B_{1}}
+\newcommand{\aphi}{C_{1}}
+\newcommand{\az}{E_{12}}
+\newcommand{\bx}{A_{2}}
+\newcommand{\bu}{B_{2}}
+\newcommand{\bg}{C_{2}}
+\newcommand{\bz}{E_{22}}
+\newcommand{\bxdot}{E_{21}}
+\newcommand{\lo}{A_{\text{LO}}}
+\newcommand{\lx}{L_x}
+\newcommand{\lxdot}{L_{\dot{x}}}
+\newcommand{\lu}{L_u}
+\newcommand{\lz}{L_z}
+
+The following dynamic system structure is called Generalized Nonlinear Static Feedback structure (GNSF), as it consists of a nonlinear static feedback (NSF) coupled to an index-1 differential algebraic equation (DAE) and a linear output system (LOS). It reads as:
+
+\begin{align*}
+E \matr{c}{\dot{x}\uind{1} \\ z} & = A x\uind{1} + B u + C \phi(\lxdot\dot{x}\uind{1}+ \lx x\uind{1} + \lz z + \lu u)  	\\
+%	&= \bx x\uind{1} + \bu u + \bg g( x\uind{1}, z, u) 	\label{eq:dyn_nlsf_dae_b} \\
+\dot{x}\uind{2} &= \lo x\uind{2} + f(\dot{x}\uind{1}, x\uind{1}, z,u) 
+\end{align*}
+
+where we denote the states as $ x = \left[ {x\uind{1}}^\top, {x\uind{2}}^\top\right]^\top \in \R^{\nx} $, $ x\uind{1} \in \R^\nxone, x\uind{2}\in\R^\nxtwo $, the algebraic variables $ z \in \R^\nz $, the controls $ u \in \R^{n_u} $, the (nonlinear) functions $ \phi: \R^{\ninx + \ninu} \rightarrow \R^{\nout}, f: \R^{2\nxone + \nz + n_u} \rightarrow \R^{\nxtwo} $, which will be referred to as \defI{nonlinearity} and \defI{linear output function} respectively. Furthermore we use the model matrices $ \lxdot,\lx \in \R^{\ninx \times \nxone} $, $ \lz \in \R^{\ninx \times \nz} $, $ \lu \in \R^{\ninu \times n_u} $ (\defI{linear input matrices}) and the matrices $ A\in \R^{(\nxone + \nz) \times \nxone}$  $ B \in \R^{(\nxone + \nz) \times n_u},
```

Comment:
What is the second term in the input size of $\phi$?

$L_u$ needs to have $n_y$ rows if you want to add $L_u u$ to the other terms forming $y$.

Reply:
thanks, forgot to update this...
I split the input of phi at some point, such that the integrator does not compute the derivatives w.r.t. the controls within the Newton iteration.
Basically, I just replaced the last `+` in line 54 with a `,`